build-tools/packages/build-cli/src/commands/check/latestVersions.ts

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { isInternalVersionScheme } from "@fluid-tools/version-tools";
+import { Args } from "@oclif/core";
+import * as semver from "semver";
+import { findPackageOrReleaseGroup, packageOrReleaseGroupArg } from "../../args.js";
+import { BaseCommand, sortVersions } from "../../library/index.js";
+
+export default class LatestVersionsCommand extends BaseCommand<typeof LatestVersionsCommand> {
+	static readonly summary =
+		"Determines if an input version matches a latest minor release version. Intended to be used in the Fluid Framework CI pipeline only.";
+
+	static readonly description =
+		"This command is used in CI to determine if a pipeline was triggered by a release branch with the latest minor version of a major version.";
+
+	static semverArg = Args.custom({
+		required: true,
+		description: "The version corresponding to the pipeline trigger branch.",
+		parse: async (input: string) => {
+			if (semver.valid(input) === null || semver.valid(input) === undefined) {
+				throw new Error(`Invalid version: ${input}. Please provide a valid SemVer version.`);
+			}
+			return input;
+		},
+	});
```

Comment:
This can be put in src/args.ts and exported so other commands can use it. I suggest doing somthing like this:

```ts
export const semverArg = Args.custom<semver.SemVer, { loose?: boolean }>({
	required: true,
	description:
		"A semantic versioning (semver) version string. Values are verified to be valid semvers during argument parsing.",
	parse: async (input, _, opts) => {
		const parsed = semver.parse(input, opts.loose);
		if (parsed === null) {
			throw new Error(`Invalid semver: ${input}`);
		}
		return parsed;
	},
});
```

That will make the type of the arg a semver.SemVer instead of a string.

Then in the command, you can use it like so:

```ts
	static readonly args = {
		version: semverArg({
			required: true,
			description: "The version corresponding to the pipeline trigger branch.",
		}),
	} as const;
```

You can override the default description/required/etc. values in the arguments.